docker-for-mac/space.md

```diff
@@ -5,90 +5,96 @@ title: Disk utilization in Docker for Mac
 ---
 
 Docker for Mac stores Linux containers and images in a single, large "disk image" file
-in the Mac filesystem. This is different from Docker on Linux, which usually stores containers
-and images in the `/var/lib/docker` directory.
+in the Mac filesystem. This is different from Docker on Linux, which usually stores containers and images in the `/var/lib/docker` directory.
 
-## Where is the "disk image" file?
+## Where is the disk image file?
 
-To locate the "disk image" file, first select the whale menu icon and then select
-**Preferences...**. When the **Preferences...** window is displayed, select **Disk** and then **Reveal in Finder**:
+To locate the disk image file, select the Docker icon and then
+**Preferences** > **Resources** > **Advanced**.
 
-![Disk preferences](images/settings-disk.png)
+![Disk preferences](images/menu/prefs-advanced.png)
```

Comment:
Looks like a bug. I've created a ticket for this. Will update the screenshot when this issue is fixed.

Reply:
Added a note about updating the screenshot in the Desktop Jira ticket (DESKTOP-1902). Suggest we go ahead with the docs changes instead of waiting until the UI is fixed.